src/test/project-manager.test.ts

```diff
@@ -0,0 +1,29 @@
+import * as chai from 'chai';
+import chaiAsPromised = require('chai-as-promised');
+chai.use(chaiAsPromised);
+const assert = chai.assert;
+import { TypeScriptService } from '../typescript-service';
+import { initializeTypeScriptService, shutdownTypeScriptService, TestContext } from './typescript-service-helpers';
+
+describe('ProjectManager', () => {
+	describe('getPackageName()', function (this: TestContext) {
+		before(async function (this: TestContext) {
+			await initializeTypeScriptService(
```

Comment:
I would prefer if this just created a ProjectManager with a mocks. Might require to move the FS mock into an extra file.